BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.1.0-preview2-02606-01
+2.1.0-preview2-02606-05
```

Comment:
FYI: This update brings in a fix for a known official build break: https://github.com/dotnet/core-eng/issues/2828. Your publish leg will probably fail until this PR is merged.

OSX CI is known to be in a bad state right now: https://github.com/dotnet/core-eng/issues/2808. I recommend merging now, since those are the only CI legs not green yet.